src/crimson/os/seastore/backref_manager.h

```diff
@@ -97,6 +97,44 @@ class BackrefManager {
 					//   created by this insertion
   ) = 0;
 
+  virtual std::set<
+    backref_buf_entry_t,
+    backref_buf_entry_t::cmp_t> get_cached_backrefs_in_range(
```

Comment:
nit, add name alias to both `std::set<backref_buf_entry_t, backref_buf_entry_t::cmp_t>` and `std::set<Cache::backref_extent_buf_entry_t, Cache::backref_extent_buf_entry_t::cmp_t>`